iib/workers/config.py

```diff
@@ -153,45 +159,66 @@ def validate_celery_config(conf, **kwargs):
     if not isinstance(conf['iib_organization_customizations'], dict):
```

Comment:
Could you move this check to a new private function? That would make it easier to read.